### ISSUE TITLE
Deep Learning (overview) and its applications in modern Natural Language Processing

### ISSUE THREAD

**Issue by yugrocks** [pydatadelhi/talks#69]
- **Abstract (2-3 lines):** In this talk, I will present an overview of Deep Learning, and then present in detail what are the advancements being made in Natural Language Processing, using deep learning.

- **Brief Description and Contents to be covered:**
    - Deep Learning overview : Types of Neural Networks and how to train them.         
    - Introduction to Natural language processing:      
          - From basic concepts (tagging, parsing, information extraction) to applications like   
            conversational interfaces, question answering systems, text summarization, word embeddings, document classification etc.
    - Previously used state of the art techniques, and how they are being transformed using Deep Learning.      
    - Where are we heading and what is the future of NLP based systems.

- **Pre-requisites for the talk:**      
    Basic Machine Learning and python

- **Time required for the talk:**
   40 minutes.

- **Link to slides**
    [here](https://slides.com/yugalsharma/deck/#/)

- **Will you be doing hands-on demo as well?**
   Probably, yes.

- **Link to ipython notebook (if any)**
   Soon.
- **About yourself**
   Btech. CSE student at Maharaja Surajmal Institute of Technology, currently researching on information extraction and document classification in NLP at Grainpad Pvt. Ltd.

- **Are you comfortable if the talk is recorded and uploaded to PyData Delhi's YouTube channel ?**
  Yes.

- **Any query ?**
  No.

**Comment by shagunsodhani** [comment]
Hey @yugrocks Thanks for another proposal! I have a couple of points:

- `Deep Learning overview` - Would this be focused on just Language Modelling tools?
- `From basic concepts (tagging, parsing, information extraction) to applications like
conversational interfaces, question answering systems, text summarization, word embeddings, document classification etc.` - Just to make sure there is no confusion, would you be also talking about how linguistics could be used for say text summarization? If yes, it would be a very nice and useful comparison :)
- For the hands-on part (if any), what framework and dataset are you planning to use?

**Comment by yugrocks** [comment]
Hi @shagunsodhani . Let me answer these one by one:

- The section about Deep Learning will just be an overview of what is deep learning and how neural networks work. The next section is dedicated to NLP, in which I'm planning to cover language modeling using Deep Learning and otherwise; and all other stuff mentioned in the second point above. 

- Yes, definitely. I am going to show a comparison of deep learning based techniques(with explanation)  **vs** all other techniques for solving all the above mentioned problems (for example- using linguistic and statistical features along with classifiers say CRFs **vs** sequence labeling using RNNs, for Named Entity Recognition)

- For the hands-on part, I shall be using the NLTK (python) ; the brown and conll2000 corpora.

**Comment by manojpandey** [comment]
Hi @yugrocks - any update on slides?
We're having the meetup this Saturday!

**Comment by yugrocks** [comment]
Hi @manojpandey , I'm working on the slides and will upload them by tomorrow (for this proposal only).

**Comment by manojpandey** [comment]
Sure @yugrocks - looking forward !

**Comment by yugrocks** [comment]
Hello @manojpandey,  [here](https://slides.com/yugalsharma/deck/#/) are the slides that I am preparing. It'll be completed by thursday evening. Till then please review it and let me know if I need to change something :)
Also the Deep Learning part is very brief, and I'm thinking of leaving it (to save time) if enough audience have little to intermediate knowledge of it.

**Comment by yugrocks** [comment]
Slides added. Please review.

**Comment by manojpandey** [comment]
Hey @yugrocks - I checked the slides. Will you also be using a notebook to show/run the code for some of the projects you listed in your slides?

Btw, the meetup is tomorrow, and this will be the second talk. Can you confirm with me or @mananpal1997 over Telegram ASAP - thanks!

**Comment by yugrocks** [comment]
No, I've decided not to do hands on for this talk because I'm afraid it may stretch the time of the talk considerably. So I'll not be using any notebook for now. 
By the way, the talk is accepted, right?   :)

**Comment by manojpandey** [comment]
Yes, second slot tomorrow.
Check https://meetup.com/PyDataDelhi for time and venue.
